scripts/deploy_to_grading.py

```diff
@@ -103,14 +103,29 @@ def _evaluate_task(taskname, repository):
     _execute_metrics(taskname, task_conf["%s_METRICS" % taskname.upper()])
     _evaluate_metrics(taskname, task_conf["%s_METRICS" % taskname.upper()], task_conf)
 
+def _create_artifact(assignment_configuration):
+    # Runs a script to collect individual metric results of every task and create
+    # an archive containing all results
+    script_path = os.path.join(os.environ["D2G_PATH"],
+        "scripts/create_artifact.sh")
```

Comment:
Warum machst Du eigentlich das Kopieren und Zip'en nicht hier mit im Python-Skript? 

Ich war anfangs verwirrt, weil bei den anderen Skripten das Shell-Skript in der GH-Action aufgerufen wird, und dieses dann den jeweiligen Python-Code triggert. Hier ist es anders herum :)

Reply:
> Warum machst Du eigentlich das Kopieren und Zip'en nicht hier mit im Python-Skript?

Weil dieses Skript nur andere Skripte anstoßen soll.

> Ich war anfangs verwirrt, weil bei den anderen Skripten das Shell-Skript in der GH-Action aufgerufen wird,

Die anderen Shell-Skripte werden auch von diesem Python-Skript aufgerufen. :)

Reply:
wtf?

Reply:
> Ich war anfangs verwirrt

Jetzt bin ich verwirrt.

Reply:
hmmm. dann mach einfach. ich sehe ja hinterher das gesamtbild.